shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -921,6 +927,42 @@ - (void)waitForFirstFrame:(NSTimeInterval)timeout
   });
 }
 
+- (FlutterEngine*)spawnWithEntrypoint:(NSString*)entrypoint {
+  assert(_shell);
+  FlutterEngine* result =
+      [[FlutterEngine alloc] initWithName:[_labelPrefix stringByAppendingString:@"-spawn"]
```

Comment:
I think as we go through the code, we're going to find a bunch of things like this that have some conceptual ambiguity we should think through in the new world. i.e. here, it seems like the only place this is used is to name the threads for instruments etc. Except we're sharing the same thread already so this isn't used. We should either get rid of it for subsequent threads and reduce the scope for the ivar name or if we keep it generic (and make it usable for someone that picks it up one day for non-thread naming purposes), come up with a scheme. i.e. are engines 2, 3, 4 all going to be named FlutterEngine-spawn? Or if we group build off of the last one, it'll be FlutterEngine-spawn, FlutterEngine-spawn-spawn etc?

In the end, this one thing is inconsequential but I think we should take the time to build this slowly and clean these things as we go.

Reply:
Yea, good point.  I forgot to revisit it when I moved it over.  Let me think if there is something better we can do now.  These strings might actually be thrown away because we are sharing a threadhost.

Reply:
Addressed in "xiao feedback 2" -- The name of an engine is just used to create thread names, it is ignored in the case of spawning so I've just passed in the _labelPrefix as to not confuse readers of the code.